src/Compilers/CSharp/Test/Semantic/SourceGeneration/AdditionalSourcesCollectionTests.cs

```diff
@@ -36,6 +36,14 @@ public class AdditionalSourcesCollectionTests
         [InlineData("abc(1).cs")]
         [InlineData("abc[1].cs")]
         [InlineData("abc{1}.cs")]
+        [InlineData("..")]
```

Comment:
>".." [](http://example.com/codeflow?start=20&length=4)

Will this result in a generated source file outside of the current directory?

Reply:
No, this will be interpreted as file name, so it will generate `...cs` file.

Reply:
can/do we have explicit tests to validate that?

Reply:
> can/do we have explicit tests to validate that?

Added to `CommandLineTests.SourceGenerators_WriteGeneratedSources`.